modules/tournament2/timetable.php

```diff
@@ -10,17 +10,19 @@
 // Generate Table-head
 $mintime = 9_999_999_999;
 $maxtime = 0;
-$tournaments = $db->qry("
-  SELECT *,
+//@TODO: ^rework these - values do not make sense in contect of functionality
+
+$tournaments = $database->queryWithFullResult(
+  "SELECT *,
```

Comment:
What do you think replacing `*` with all fields from the `tournament_tournaments` table?

I prefer an explicit listing. Mainly because it enables searching for the usage of a database field. It's not a merge blocker, but I think we should make this codebase more explicit.

Reply:
Done. It was a bit more complicated as some fields are implicitly used by `$tfunc->GetGameEnd()`, but that was solved by a bit of pen&paper ;)